docs/running-nakama/advanced.adoc

```diff
@@ -0,0 +1,98 @@
+= Operate Nakama
+
+Day to day operation of Nakama is straightforward, requiring minimal intervention. There are just three `nakama` commands that you'll need.
+
+== nakama migrate
+
+When you first run Nakama you need to prepare the schema of the data that Nakama will store in CockroachDB. Similarly, when a new Nakama version is released, you need to migrate the data schema to that of the new version. In both situations you use the `nakama migrate` sub-command.
+
+`nakama migrate` has four options:
+
+* `nakama migrate up` creates and updates the database schema to the latest version required by nakama.
+* `nakama migrate down` downgrades the database schema to the version requested.
+* `nakama migrate redo` rebuilds the schema from the first instance it was created.
+* `nakama migrate status` provides you details of the database schema in relation to the `migrate` subcommand.
+
+If you do not provide an option to `nakama migrate` it will do nothing.
+
+*XXX:* Is the above correct?
+
+You can also provide flags to change the behaviour of how `nakama migrate` works, these flags are:
+
+* `--db`: the location of the CockroachDB instance to use.
+** Default value is `root@localhost:26257/nakama`.
+* `--limit` Number of migrations to use when running either `up`, `down`, or `redo`.
+** Default value is `-1`.
+
+== nakama doctor
+
+Nakama can provide you with diagnostic information, which is particularly useful when you're requesting support or otherwise looking to diagnose an issue.
+
+Running `nakama doctor` generates a report that details the server's configuration and environment. If you want to diagnose a Nakama instance running on another machine, or one of many on your local machinem you can use the following flags:
+
+* `--host`: the host running the Nakama instance you want to diagnose
+** Default value is `127.0.0.1`
+* `--port` the dashboard port used by nakama, as opposed to the port used to by game clients to connect to the server.
+** Default value is `7351`
+
+Here's an example of the output from `nakama doctor`:
+
+[source,bash]
+----
+2017/04/25 14:10:54 host: 127.0.0.1
+2017/04/25 14:10:54 port: 7351
+arch: amd64
+cpus: 4
+go: go1.8
+os: linux
+version: 0.12.1+aab3e2e
+config:
+---
+database:
+  max_idle_conns: 0
+  conn_max_lifetime_ms: 60000
+  max_open_conns: 0
+social:
+  steam:
+    app_id: 0
+    publisher_key:
+ops_port: 7351
+dsns: [root@cockroachdb:26257]
+port: 7350
+session:
+  encryption_key: defaultencryptionkey
+  token_expiry_ms: 60000
+transport:
+  pong_wait_ms: 10000
+  ping_period_ms: 8000
+  server_key: defaultkey
+  max_message_size_bytes: 1024
+  write_wait_ms: 5000
+name: nakama
+data_dir: /nakama/data
+---
+
+----
+
+== nakama admin
```

Comment:
I think let's not document `nakama admin` for now as the usage for it is not official for.